docker/server-jetty/src/main/server-jetty/requirements.txt

```diff
@@ -2,6 +2,7 @@ adbc-driver-manager==1.1.0
 adbc-driver-postgresql==1.1.0
 connectorx==0.3.3; platform.machine == 'x86_64'
 deephaven-plugin==0.6.0
+docstring_parser==0.16
```

Comment:
Any new dependencies to the base images need to be made externally first: https://github.com/deephaven/deephaven-server-docker/blob/main/contexts/server-base/requirements.txt, and then an image bumping process to pull in the specific docker images. Feel free to reach out for more info.